app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<!-- <h1>Tasks#index</h1>
+<p>Find me in app/views/tasks/index.html.erb</p> -->
+
+<h1>Task List</h1>
+
+<div class="right_tasks">
+  <ul class="tasks">
+    <h4 class="task_header">Your Tasks</h4>
+
+    <% counter = 1 %>
+    <% @tasks.each do |task| %>
+    <li>
+      <!-- <%= task.name %> -->
+
+      <div class="task_name">
+        <%= "#{counter}. " %> <%=link_to(task.name, task_path(task.id)) %>
```

Comment:
instead of using a counter variable, you could probably just make this list an ordered list (<ol> instead of <ul>) and leave your list-style style alone (remove line 89 on tasks.scss)